src/ros_commo.py

```diff
@@ -293,6 +293,10 @@ def language_perceived_text_cb(self, text_heard):
 	# The chat_heard message is of type chatbot/ChatMessage
 	# from chatbot.msg import ChatMessage
 	def chat_perceived_text_cb(self, chat_heard):
+		if 'shut up' in chat_heard.utterance.lower():
+			self.tts_control_pub.publish("shutup")
```

Comment:
Ughhhh. This is a real hack.  the correct long-term solution is to run this through opencog ... so that we can handle synonymous prhases, such as "stop taking please", "hush", "quiet", "that's enough"  -- or maybe even large sudden movements or loud noises make her stop talking...